src/encoded/schemas/functional_characterization_experiment.json

```diff
@@ -236,36 +236,25 @@
                 }
             ]
         },
-        "gRNA_tiling_loci": {
-            "title": "gRNA tiling loci",
-            "description": "The loci targeted by gRNAs.",
-            "type": "array",
-            "uniqueItems": true,
-            "minItems": 2,
-            "items": {
-                "title": "Locus",
-                "description": "Gene locus targeted by gRNAs.",
-                "type": "string",
-                "linkTo": "Gene"
-            }
-        },
         "examined_loci": {
             "title": "Examined loci",
             "description": "The loci whose expression or product is under investigation for the functional characterization experiment.",
             "type": "array",
             "uniqueItems": true,
```

Comment:
I think it should be minItem=1. Otherwise, this could be an empty array - that is not useful and confusing.

Reply:
I added minItems: 1 back in.